tests/Commands/ToggleDone.test.ts

```diff
@@ -32,11 +33,20 @@ function testToggleLine(inputWithCursorMark: string, expectedWithCursorMark: str
     expect(input.length).toEqual(inputWithCursorMark.length - 1);
     expect(expected.length).toEqual(expectedWithCursorMark.length - 1);
 
+    const cursorPosition = (s: string): EditorPosition => {
+        // Split input string on cursor marker, and make array of lines
+        const linesBeforeCursor = s.split(cursorMarker, 1)[0].split('\n');
+        // Cursor was positioned at the end of the last line
```

Comment:
This is my one-and-only comment after all the great improvements you've made...

I am unsure what to do with the information 'Cursor was positioned at the end of the last line'.
How do we know that? what does it mean?
I'm probably missing something obvious, and I know that clear comments are super hard to write!

Reply:
Thank you!